src/components/Payment-steps/steps/Account.tsx

```diff
@@ -0,0 +1,268 @@
+/* eslint-disable */
+import React, { useState, useEffect, useContext } from 'react';
+import { useStepperContext } from '../contexts/StepperContex';
+import { useTranslation } from 'react-i18next';
+import { Icon } from '@iconify/react';
+import ErrorDisplay from '../ErrorDisplay';
+
+const Account = ({ onClickState }: any) => {
+  const { t } = useTranslation();
+  const [onClickCard, setOnClickCard] = useState({
+    creditCard: false,
+    paypal: false,
+    applepay: false,
+    onClickState: false,
+  });
+  const { userData, setUserData, formValidation, displayError, valid } =
+    useStepperContext();
+
+  console.log(userData);
```

Comment:
Can we remove the unnecessary consoles here when push to remote

Reply:
I thought we have hooks to block `console` messages when pushing?

Reply:
> I thought we have hooks to block `console` messages when pushing?

ESLint would have blocked it but there's seems to be `eslint-disable`